src/main/java/io/quarkus/bot/release/ReleaseInformation.java

```diff
@@ -56,14 +62,24 @@ public String getFullBranch() {
         return branch;
     }
 
+    /**
+     * @return the branch from which we branched - it is main except when preparing the CR1 of an LTS release, in which case it
+     *         is the previous version branch as we want to branch from the previous version branch
+     */
     public String getOriginBranch() {
         return originBranch;
     }
 
+    /**
+     * @return the qualifier (Alpha1, CR1, CR2...)
+     */
     public String getQualifier() {
         return qualifier;
     }
 
+    /**
+     * @return whether it's a maintenance release, meaning it's not the current branch we are releasing
```

Comment:
This doesn't make it very clear to me - is it that there already exists a branch with higher minor version?

Reply:
I added some clarification and an example, HTH :).